packages/form-js-viewer/assets/form-js.css

```diff
@@ -4,6 +4,7 @@
 .fjs-container {
   --color-grey-225-10-15: hsl(225, 10%, 15%);
   --color-grey-225-10-35: hsl(225, 10%, 35%);
+  --color-grey-225-10-40: hsl(225, 10%, 45%);
```

Comment:
```suggestion
  --color-grey-225-10-45: hsl(225, 10%, 45%);
```

😄

Reply:
Actually, this is on purpose :D

45 doesn't look right :trollface: 

But the gradient still does
![image](https://user-images.githubusercontent.com/17801113/207037527-f44be788-a81e-4fe0-a9f3-fb67da35701b.png)

Reply:
I mean, if the color lightness is `45%` then the variable name should also reflect this. Otherwise it's confusing.

Reply:
Oh you're right, I guess I got myself confused then.